src/Splat/Platforms/Android/Bitmaps/PlatformBitmapLoader.cs

```diff
@@ -44,7 +48,8 @@ public async Task<IBitmap> Load(Stream sourceStream, float? desiredWidth, float?
 
             if (desiredWidth == null)
             {
-                bitmap = await Task.Run(() => BitmapFactory.DecodeStream(sourceStream)).ConfigureAwait(true);
+                sourceStream.Position = 0;
```

Comment:
You may want to move that `sourceStream.Position = 0` to outside the if statement. I suspect both cases may benefit from it being set back to the beginning.